var/spack/repos/builtin/packages/freetype/package.py

```diff
@@ -47,14 +47,13 @@ def headers(self):
         headers.directories = [self.prefix.include.freetype2]
         return headers
 
-    def configure_args(self):
-        args = [
-            "--with-brotli=no",
-            "--with-bzip2=yes",
-            "--with-harfbuzz=no",
-            "--with-png=yes",
-            "--with-zlib=no",
+
+class CMakeBuilder(CMakeBuilder):
+    def cmake_args(self):
+        return [
+            self.define("FT_DISABLE_ZLIB", True),
+            self.define("FT_DISABLE_BROTLI", True),
+            self.define("FT_DISABLE_HARFBUZZ", True),
+            self.define("FT_DISABLE_PNG", True),
+            self.define("FT_WITH_BZIP2", True),
         ]
-        if self.spec.satisfies("@2.9.1:"):
-            args.append("--enable-freetype-config")
```

Comment:
It seems the CMake version is missing `freetype-config`. We should keep the same behavior.

Reply:
Perhaps you can elaborate here as I'm not particularly familiar with Freetype's build system but would that not be the flag for generating the config files that CMake generates by default?  Otherwise I cannot find a reference to the config files in the CMake interface. If this is indeed not a true paralell, you can disregard my comment regarding dropping autotools altogether, as you're right we don't want to drop functionality.

Reply:
This flag causes a `freetype-config` binary to be generated that can be used similarly to `pkg-config freetype`. I think some of Freetype's dependents might need it.

Reply:
After digging into things a bit, CMake properly exports this project and produces a freetype config that in general should be sufficient for dependent projects, but I found a few that truly required the autotools style config that can be treated like a binary so I'm adding back the autotools build. When/If those packages are ever ported to be solely cmake projects, this can be just a cmake package, but until then, autotools will need to be an option on unix.

Reply:
So feel free to disregard my "removing autotools entirely" comment and related thoughts, as they are wrong.